api/api/src/test/scala/uk/ac/wellcome/platform/api/works/v2/ApiV2WorksAggregationsTest.scala

```diff
@@ -55,22 +55,22 @@ class ApiV2WorksAggregationsTest extends ApiV2WorksTestBase {
                     },
                     {
                       "data" : {
-                        "id" : "d",
-                        "label" : "Journals",
+                        "id" : "k",
```

Comment:
The ordering of these aggregations changed because the terms aggregation sorts by frequency